persistent-mysql/Database/Persist/MySQL.hs

```diff
@@ -237,6 +237,7 @@ instance MySQL.Param P where
       MySQL.Plain $ BBB.fromString $ show (fromRational r :: Pico)
       -- FIXME: Too Ambiguous, can not select precision without information about field
     render (P (PersistDbSpecific s))    = MySQL.Plain $ BBS.fromByteString s
+    render (P (PersistLiteral l))     = MySQL.Plain $ BBS.fromByteString l
```

Comment:
I’m confused how this can be the same implementation as PersistDbSpecific

Reply:
see comment below https://github.com/yesodweb/persistent/pull/1122#issuecomment-689707949